src/lib/palette.styl

```diff
@@ -32,13 +32,16 @@ getColor(name)
     error('cannot find: ' + name)
   value
 
-getSpace(indices...)
+getSpace(keys...)
   values = ()
-  for i in indices
-    value = THEME.space[i]
-    if value == null
-      error('cannot find: ' + i)
-    push(values, value)
+  for key in keys
+    if key == 0 || key == 'auto'
```

Comment:
Is this change is to accommodate things like  `getSpace('auto', 1)`?

Reply:
Yep, or `0` — I was surprised to learn `'0': 0`, wasn't actually in the tokens.